### PR TITLE
fix: npm package relative paths on the browser

### PR DESCRIPTION
**Pull request by llorllale**
* Fixed npm package relative paths on the browser
   * Note: the code assumes the assets are under `/aries-framework-go`.
* Also added a basic Vue.js sample application.

Signed-off-by: George Aristy <goerge.aristy@securekey.com>

### CODE REVIEW

File: cmd/aries-js-worker/src/aries.js [hunk 1]
```diff
@@ -6,7 +6,24 @@ SPDX-License-Identifier: Apache-2.0
...
+
+// base path to load assets from at runtime
+const __publicPath = _ => {
+    if (inNode) {
+        // TODO determine module_path at runtime
+        return process.cwd() + "/node_modules/@hyperledger/aries-framework-go/"
+    } else if (inBrowser) {
+        return "/aries-framework-go/"
+    } else {
+        // TODO throw error?
```
Comment (troyronda) [inline cmd/aries-js-worker/src/aries.js]: Issue?

File: cmd/aries-js-worker/vue-aries-framework-go/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (troyronda) [inline cmd/aries-js-worker/vue-aries-framework-go/package.json]: Do we need lock file checked in? Suggest omitting for now if we can.

File: cmd/aries-js-worker/vue-aries-framework-go/public/index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!--
```
Comment (troyronda) [inline cmd/aries-js-worker/vue-aries-framework-go/public/index.html]: Please remove favicon.

File: cmd/aries-js-worker/vue-aries-framework-go/package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  "name": "vue-aries-framework-go",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "bash scripts/serve.sh",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@hyperledger/aries-framework-go": "^0.1.2-snapshot-cd14bfe",
```
Comment (troyronda) [inline cmd/aries-js-worker/vue-aries-framework-go/package.json]: TODO: find better way.

File: cmd/aries-js-worker/vue-aries-framework-go/src/components/HelloFrameworkGo.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<!--
```
Comment (troyronda) [inline cmd/aries-js-worker/vue-aries-framework-go/src/components/HelloFrameworkGo.vue]: Please remove logo.png

**Comment by rolsonquadras** [comment]
@llorllale @troyronda Just wondering if the sample app to test the JS wrapper should be inside `cmd` folder. May be inside test or a new example folder at higher level would seem better.

File: cmd/aries-js-worker/vue-aries-framework-go/package.json (hunk 4 above)
Comment (llorllale) [reply cmd/aries-js-worker/vue-aries-framework-go/package.json]: #1228

File: cmd/aries-js-worker/vue-aries-framework-go/src/components/HelloFrameworkGo.vue (hunk 5 above)
Comment (llorllale) [reply cmd/aries-js-worker/vue-aries-framework-go/src/components/HelloFrameworkGo.vue]: Removed. However, the app should have one or two icons at some point (even if it's just a sample app) - can't imagine it being "pretty" without any.

**Comment by llorllale** [comment]
@troyronda @rolsonquadras I'm going to merge this to move faster. Feel free to suggest an appropriate path for vue-aries-framework-go. Note that the rest api stuff also resides in cmd.

**Review by troyronda — APPROVED** [review]

**Comment by rolsonquadras** [comment]
> @troyronda @rolsonquadras I'm going to merge this to move faster. Feel free to suggest an appropriate path for vue-aries-framework-go. Note that the rest api stuff also resides in cmd.
> 
> Edit: chucks, I can't approve my own PR

Do we have an issue to track this ? We can discuss on the path later.

**Pull request merged**